<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weight_window</class>
 <widget class="QMainWindow" name="Weight_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="weight_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Weight Conventer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="from_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>from</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="to_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 187, 255);</string>
    </property>
    <property name="text">
     <string>Convert Into</string>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>20</width>
      <height>291</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="value_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="answer_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 177, 138);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>221</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="kg_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Kilogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gram_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Gram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="milli_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Milligram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="metric_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Metric Ton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="long_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Long Ton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="short_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Short Ton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pound_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Pound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ounce_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Ounce</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="car_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Carrat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="atom_button">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 255, 155);</string>
       </property>
       <property name="text">
        <string>Atomic Mass Unit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>221</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="kg_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Kilogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gram_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Gram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="milli_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Milligram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="metric_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Metric Ton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="long_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Long Ton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="short_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Short Ton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pound_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Pound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ounce_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Ounce</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="car_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Carrat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="atom_button_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 206, 154);</string>
       </property>
       <property name="text">
        <string>Atomic Mass Unit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
